cmd/cloud/server.go

```diff
@@ -227,3 +233,28 @@ func deprecationWarnings(logger logrus.FieldLogger, cmd *cobra.Command) {
 		logger.Warn("[Deprecation] Flag certificate-aws-arn is deprecated. Provisioner will use a resource tag to find certificate-aws-arn in AWS.")
 	}
 }
+
+// getHumanReadableID  represents  a  best  effort  attempt  to  retrieve  an
+// identifiable  human to  associate with  a given  provisioner. Since
+// this is for dev workflows, any  error causes it to merely return a
+// generic string.
+func getHumanReadableID() string {
+	cmd := exec.Command("git", "config",
+		"--get", "user.email")
+
+	envVar := os.Getenv("CLOUD_SERVER_OWNER")
+	if envVar != "" {
+		return envVar
+	}
+
+	output, err := cmd.Output()
+	if err != nil {
+		logger.Debugf("Couldn't determine username of developer with which to tag infrastructure due to error: %s", err.Error())
```

Comment:
nit: hmm, which logger is this using? One isn't passed in. As long as it works though it's not a big deal.

Reply:
Here we're in package `main` so this is the global logger from `logger.go`. Since this is just a helper function I think that makes sense, is the correct context, and I don't think initializing one with new fields will buy us anything.